build_files/packages-dx.sh

```diff
@@ -20,20 +20,23 @@ EXCLUDED_PACKAGES=($(jq -r "[(.all.exclude | (select(.all != null).all)[]), \
                     (select(.\"$FEDORA_MAJOR_VERSION\" != null).\"$FEDORA_MAJOR_VERSION\".exclude | (select(.dx != null).dx)[])] \
                     | sort | unique[]" /tmp/packages.json))
 
+# store a list of RPMs installed on the image
+INSTALLED_EXCLUDED_PACKAGES=()
+
 # ensure exclusion list only contains packages already present on image
 if [[ "${#EXCLUDED_PACKAGES[@]}" -gt 0 ]]; then
-    EXCLUDED_PACKAGES=($(rpm -qa --queryformat='%{NAME} ' ${EXCLUDED_PACKAGES[@]}))
```

Comment:
The list of EXCLUDED_PACKAGES is being overwritten here to be only those installed on the image before installing any packages (therefore does not include gnome-extensions-app).